packages/textfield/stories/textarea.stories.ts

```diff
@@ -75,9 +75,29 @@ export const Default = (): TemplateResult => {
 export const quiet = (): TemplateResult => html`
     <sp-field-label for="story">Enter your life story...</sp-field-label>
     <sp-textfield
+        multiline
+        id="story"
+        quiet
+        placeholder="Enter your life story"
+    ></sp-textfield>
+`;
+
+export const defaultAutofocus = (): TemplateResult => html`
+    <sp-field-label for="story">Enter your life story...</sp-field-label>
+    <sp-textfield
+        multiline
+        id="story"
         autofocus
+        placeholder="Enter your life story"
+    ></sp-textfield>
+`;
+
+export const quietAutofocus = (): TemplateResult => html`
```

Comment:
The default quiet story looks good now- thanks for adding these extra autofocus stories!